apps/wildfires/src/app/shared/components/importantTips/ImportantTips.tsx

```diff
@@ -0,0 +1,49 @@
+import { CameraIcon, InfoIcon } from '@monorepo/react/icons';
+
+export default function ImportantTips() {
+  return (
+    <div className="bg-brand-yellow-light rounded-[20px] p-4 md:p-8 md:pb-[3.75rem] w-full my-12 md:my-20">
+      <h3 className="uppercase font-bold md:text-2xl mb-8 md:mb-10">
+        Important Tips
+      </h3>
+      <h2 className="font-bold text-2xl md:text-[40px] md:leading-[52px] pl-6 border-l-[10px] border-brand-orange mb-4 md:mb-[3.875rem]">
+        Tax & Insurance
+      </h2>
+      <div className="bg-white rounded-[20px] w-full p-6 mb-4 md:mb-10">
+        <div className="flex items-center gap-4 mb-6">
+          <InfoIcon className="h-6 min-w-6 md:h-8 md:min-w-8" />
+          <h3 className="font-bold text-xl md:text-2xl">
+            Crowdfunding and Tax Considerations
```

Comment:
Would it be a lot of work to create a reusable component out of this? It looks like the Emergency Alerts section below will need the same component. Actually, I don't see a ticket for that section. Do you want to create a ticket for it or update DEV-1274 to include that section?